src/helpers/_focusable.scss

```diff
@@ -32,3 +32,38 @@
     background-color: $govuk-focus-colour;
   }
 }
+
+/// Focusable with box-shadow
+///
+/// Removes the visible outline and replace with box-shadow and background colour.
+/// Used for interactive text-based elements.
+///
+/// @access public
+
+@mixin govuk-focusable-text {
+  // When colours are overridden, for example when users have a dark mode,
+  // backgrounds and box-shadows disappear, so we need to ensure there's a
+  // transparent outline which will be set to a visible colour.
+
+  // Since Internet Explorer 8 does not support box-shadow, we want to force the user-agent outlines
+  @include govuk-not-ie8 {
+    outline: $govuk-focus-width solid transparent;
+    outline-offset: 0;
+  }
+  color: $govuk-text-colour;
+  background-color: $govuk-focus-colour;
+  box-shadow:
```

Comment:
Very much personal preference, but I'd probably disable the linter for this line and align the values:

```
box-shadow: -5px -1px 0 1px govuk-colour("yellow"),
             5px -1px 0 1px govuk-colour("yellow"),
            -3px  1px 0 3px govuk-colour("black"),
             3px  1px 0 3px govuk-colour("black");
```

It feels like this could do with a comment as well, but I don't even know where to start trying to explain how all these shadows interact with each other… 🤪

Reply:
Yeah lets do this